client/asset/btc/spv.go

```diff
@@ -211,24 +204,6 @@ func (w *btcSPVWallet) Start() (SPVService, error) {
 	w.chainClient = chain.NewNeutrinoClient(w.chainParams, w.cl)
 	w.Wallet = btcw
 
-	oldBday := btcw.Manager.Birthday()
```

Comment:
`updateDBBirthday` is no longer used. Can be deleted in the btc, bch, and ltc wallets.